webhelper/README

```diff
@@ -26,9 +26,9 @@ Prominent features
 Requirements
 ============
 
-This plugin requires GTK+ (>= 2.16), GLib (>= 2.16), GIO (>= 2.18),
-GdkPixbuf (>= 2.0), WebKitGTK (>= 1.1.18), and obviously Geany (>= 0.20) to
-work. If you intend to build it yourself, you will need to get the development
+This plugin requires GTK3, GLib (>= 2.22), GIO (>= 2.18), GdkPixbuf (>= 2.0),
+WebKit2GTK (either API 4.0 or 4.1), and obviously Geany (>= 0.20) to work.
```

Comment:
Geany 0.20 or 2.0?

Reply:
Who knows?  As mentioned, I didn't properly review those numbers just yet.  Geany dep is likely to be closer to 0.20 than 2.0 though.  Others like GTK3 or GLib might be higher.
But I don't plan in fiddling with that for the moment, because it's less trivial than what I feel it gains, and can be done at a later point.

Reply:
I thought it might have been a typo 0.2 vs 2.0 ;-) 

Don't forget that the plugin READMEs go to the plugins.geany.org site, probably doesn't have to be 2.0 but pretending it will work with any Geany pre-GTK3 is wrong.

Reply:
> but pretending it will work with any Geany pre-GTK3 is wrong.

Good point.  Well, what about you opening a "update plugins requirements" PR after having checked and updated them all? :grin: 
Though, if I find the time (maybe tonight? who knows) I'll try and do that for this one.  But I don't love seeing this endlessly delayed for basically unrelated stuff… although I laid that on myself given the PR title, didn't I?

Reply:
Well, you updated requirements in README as part of #1336, which is what prompted me to mention it here and on the markdown PR, so its all your own fault :grin: 

And you had already made changes to README here so it seemed pretty simple to add.

Reply:
Damned :smile: 

Simple to add, yes, but simple to figure out, not so much.
I sometimes wish there was a generic tool that can *reliably* scan the code an tell me "ok, you need libfoo >= 1.2.3 (because of $reason), libbar 42.0 (because of $othereason), …", but well.  I had a script that did something like that ad-hoc grepping GTK documentation and the code to try and get an idea, but not sure if it still works, and it never was truly reliable.

Reply:
What was the first version of Geany supporting GTK 2.x?  Is it necessary to mention Geany version?

Reply:
Well, do we need to list all those things individually?  I would expect a new enough webkit2gtk will require newer versions of Glib, GTK3, etc than the plugin.  So wherever a builder gets their webkit2gtk-dev from should have the right parts automatically ... won't it?

Reply:
Probably